runtime/ops/http.rs

```diff
@@ -78,6 +82,23 @@ fn op_http_start(
   Err(bad_resource_id())
 }
 
+#[op]
+fn op_flash_upgrade_http(
```

Comment:
Why does this op has to be here?

Reply:
Because op_http_upgrade is here too (circular dependency on TcpStreamResource)